### PR TITLE
Initial Merge of ApplicationInsights-JS repo and cleanup

### PR DESCRIPTION
**Pull request by MSNev**
Initial Cleanup to remove unused files from this repo
Rework README.md, RELEASES.md
Remove extension sub-folder
Update Build steps to handle removal of extension folder

### CODE REVIEW

**Comment by MSNev** [comment]
Will be waiting for internal Open Source approval process to complete before merging

**Review by hectorhdzg — APPROVED** [review]

**Pull request merged**
